outputs.tf

```
@@ -12,6 +12,11 @@ output "secret_crn" {
   value       = local.secret_crn
 }
 
+output "secret_group_id" {
+  description = "Secret group ID of the created Secret"
```

Uppercase not needed in the word "secret"
```suggestion
  description = "Secret group ID of the created secret"
```

---

@alex-reiff very minor comment, but maybe you missed it